src/main/java/org/cactoos/scalar/ItemAt.java

```diff
@@ -139,63 +142,42 @@ public ItemAt(final Iterable<T> iterable, final T fallback) {
      * @param fallback Fallback value
      */
     public ItemAt(
-        final Iterator<T> iterator,
-        final Func<Iterable<T>, T> fallback
+        final Func<Iterable<T>, T> fallback,
```

Comment:
constructor `ItemAt (fallback, iterator)` needs to be covered by a unit test

Reply:
@ilyakharlamov since I also had to do #1104, I simply removed this constructor instead.